Library/Formula/pla.rb

```diff
@@ -0,0 +1,32 @@
+class Pla < Formula
+  desc "A tool for building gantt charts in format PNG, EPS, PDF or SVG."
+  homepage "http://www.arpalert.org/pla.html"
+  url "http://www.arpalert.org/src/pla-1.1.tar.gz"
+  sha256 "a213c5e1060c97618d0eb0462cfdf738532591b30f18b36a37c1a4398346ac37"
+
+  depends_on "cairo"
+
+  def install
+    ENV.prepend "CFLAGS", "-I#{Formula["cairo"].opt_include}/cairo"
```

Comment:
Are you sure this is needed? It shouldn't be as `superenv` should handle this for you.